CONTRIBUTING.md

```diff
@@ -66,10 +66,14 @@ The following section explains various suggestions and procedures to note during
 
 ### Prerequisites
 
-* It is strongly recommended that you use Linux distributions systems or OSX for development.
+* It is strongly recommended that you use Linux distributions systems or macOS for development.
+* Running [WSL 2 (on Windows)](https://learn.microsoft.com/en-us/windows/wsl/) is also possible. Note that if during development you run a local Kubernetes cluster and have a Service with `service.spec.sessionAffinity: ClientIP`, it will break things until it's removed[^windows_xt_recent].
```

Comment:
This sounds like a common problems section

Reply:
@squat do you think it's worth it to have a common problems just for Windows? I thought about it but seems like too much without other problems, but I'm happy to add it if it could be useful.

Reply:
:+1: maybe it's premature optimization then

Reply:
@squat so are we good to go without this extra "common problems" section? 😃

Reply:
Yes 👍